met-web/src/services/contactService/index.tsx

```diff
@@ -0,0 +1,52 @@
+import http from 'apiManager/httpRequestHandler';
+import Endpoints from 'apiManager/endpoints';
+import { replaceUrl } from 'helper';
+import { Contact } from 'models/contact';
+
+export const getContact = async (contactId: number): Promise<Contact> => {
+    const url = replaceUrl(Endpoints.Contacts.GET, 'contact_id', String(contactId));
+    if (!contactId || isNaN(Number(contactId))) {
+        return Promise.reject('Invalid Contact Id ' + contactId);
```

Comment:
its defined a number already..so not sure u need to check is Nan...
if you are planning to keep it because there are cases where contactid could be soemthing else, do it before u get the url ;as the first line

Reply:
agreed, unnecessary, removed.